mindbender/maya/pythonpath/userSetup.py

```diff
@@ -2,6 +2,50 @@
 
 from maya import cmds
 
+import os
+
+
+def mayafpsconverter(Sfps):
```

Comment:
Ah no you are right, it should be in `mindbender.maya.lib` in that case. But personally I would just put it [here](https://github.com/mindbender-studio/core/pull/41/files#diff-59cde0eedfb004dc984ab1e8d2b8bd9aR67) and skip the function. It's so short and specific to this task.